frontend/model/contracts/group.js

```diff
@@ -548,7 +548,8 @@ sbp('chelonia/defineContract', {
             groupCreator: meta.username,
             distributionPeriodLength: 30 * DAYS_MILLIS,
             inviteExpiryOnboarding: INVITE_EXPIRES_IN_DAYS.ON_BOARDING,
-            inviteExpiryProposal: INVITE_EXPIRES_IN_DAYS.PROPOSAL
+            inviteExpiryProposal: INVITE_EXPIRES_IN_DAYS.PROPOSAL,
+            publicChannelCreateAllowance: false
```

Comment:
`publicChannelCreateAllowance` is bad English. Could you please rename this everywhere to something like, `allowPublicChannels` ?